Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -107,6 +120,11 @@ class TextureCacheBase
     u32 GetNumLevels() const { return texture->GetConfig().levels; }
     u32 GetNumLayers() const { return texture->GetConfig().layers; }
     AbstractTextureFormat GetFormat() const { return texture->GetConfig().format; }
+    u32 InMemoryFormat() const { return format & 0xff; }
```

Comment:
To clarify, there have always been 6 bit formats, but the two extra bits were only used during efb copies.

Until hybrid, we could guarantee that tcache entries would be using one of the 4bit formats.